test/test-sessions.js

```diff
@@ -72,4 +76,44 @@ describe('sessions', function () {
         });
     });
   });
+
+  describe.only("#pluckSession()", function () {
```

Comment:
@LoganVoisin @wilsuriel03 Notice how we just reused tests and some of the test data in `dummyData.js` I highly recommend taking this approach and build out your tests first before solving solutions.